library/src/main/java/com/parse/twitter/Twitter.java

```diff
@@ -109,6 +109,13 @@ public void setScreenName(String screenName) {
     this.screenName = screenName;
   }
 
+  private HttpURLConnectionClient getHttpClient() {
+    if (httpURLConnectionClient == null) {
+      httpURLConnectionClient = HttpURLConnectionClient.create();
+    }
+    return httpURLConnectionClient;
+  }
```

Comment:
We'd want to synchronize this to protect against it being executed asynchronously from multiple threads.